From 43c90d912e2b13e106f98af449f2a9ce94ff19f4 Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Tue, 13 Mar 2018 13:06:21 +0000
Subject: [PATCH 2/3] Settings: Hide Night Mode suggestion if LiveDisplay
 feature is present

 * Workaround by forcebly reporting that suggestion has been completed,
   when LiveDisplay is available.

Change-Id: I01f6b3a5acfeeca5468a9926b0bc21247a458dbf
---
 .../settings/dashboard/suggestions/SuggestionFeatureProviderImpl.java  | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/com/android/settings/dashboard/suggestions/SuggestionFeatureProviderImpl.java b/src/com/android/settings/dashboard/suggestions/SuggestionFeatureProviderImpl.java
index 347e165093..2d822af24d 100644
--- a/src/com/android/settings/dashboard/suggestions/SuggestionFeatureProviderImpl.java
+++ b/src/com/android/settings/dashboard/suggestions/SuggestionFeatureProviderImpl.java
@@ -166,7 +166,8 @@ public class SuggestionFeatureProviderImpl implements SuggestionFeatureProvider
     @VisibleForTesting
     boolean hasUsedNightDisplay(Context context) {
         final ContentResolver cr = context.getContentResolver();
-        return Secure.getInt(cr, Secure.NIGHT_DISPLAY_AUTO_MODE, 0) != 0
+        return context.getPackageManager().hasSystemFeature("org.lineageos.livedisplay")
+                || Secure.getInt(cr, Secure.NIGHT_DISPLAY_AUTO_MODE, 0) != 0
                 || Secure.getString(cr, Secure.NIGHT_DISPLAY_LAST_ACTIVATED_TIME) != null;
     }
 }
-- 
2.15.1

